code_generation/class.h

```diff
@@ -284,6 +284,15 @@ class KODE_EXPORT Class
      */
     bool hasFunction(const QString &name) const;
 
+    /**
+     * Sets whether the Q_OBJECT macro should be generated at the first private
+     * section. The Q_OBJECT macro will be generated if any signals or slots
+     * added to the class, so this function is mainly useful when generating
+     * intermediate classes derived from QObject, but not having any added signals
+     * or slots.
+     */
+    void setQObject(const bool isQObject);
```

Comment:
The `const` here serves no purpose, and I fear some compilers might warn about that.

Reply:
Makes sense, fixed.